power/nut-client/files/group

```diff
@@ -0,0 +1,2 @@
+dialout:x:20:nut
```

Comment:
do we need all the `group` `gshadow` and `shadow` files, doesn't `passwd` only suffice?

Reply:
`shadow` and `gshadow` aren't needed (left over from debugging).  `group` is needed as configure/upsmon use group names, not gid